.github/workflows/note-arduino-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Note Tutorials CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  validate_esp32_examples: # job id
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        include:
+          - sketch: sensor-tutorials/arduino-wiring/nano-no-library
```

Comment:
Perhaps this matrix entry should be pruned as covered by: [`note-arduino`: Example 0 - Libraryless Communication](https://github.com/blues/note-arduino/tree/master/examples/Example0_LibrarylessCommunication)

_see details below..._